[daggy-u] [dbt] - Add Lesson 7 (DEV-55) (#20096)
dagster-io/dagster: docs/dagster-university/pages/dagster-dbt/lesson-7/3-setting-up-dagster-cloud.md

    - `.github/workflows/branch_deployments.yml` - This file enables the use of [Branch Deployments](https://docs.dagster.io/dagster-cloud/managing-deployments/branch-deployments), a Dagster Cloud feature that automatically creates staging environments for your Dagster code with every pull request. We won’t work with Branch Deployments during this lesson, but we highly recommend trying them out!

---

## Checking deployment status

It looks like the deployment was completed, but it failed. If we look in the GitHub Action logs for the job, we’ll see the following error in the **Python Executable Deploy** step:

```yaml
Error: Some locations failed to load after being synced by the agent:
Error loading dagster_university: {'__typename': 'PythonError', 'message': "FileNotFoundError: [Errno 2] No such file or directory: '/venvs/3eca07cc1eb5/lib/python3.8/site-packages/working_directory/root/analytics/target/manifest.json'\n" ...
```

> It looks like the deployment failed because Dagster could not find a dbt manifest file. In the next section of this lesson, we’ll walk you through fixing this.

Reviewer: It looks like we say "it looks like" a lot. Proposing an alternative wording.
```suggestion
Your deployment failed because Dagster could not find a dbt manifest file. In the next section of this lesson, we’ll walk you through fixing this.
```
